src/App.js

```diff
@@ -32,22 +36,26 @@ function App() {
 
   return (
     <div className="App">
-      <button onClick={play}>Boop!</button>
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <div id="top">
+        <img id="logo" src="/logo.svg" alt="Logo"/>
+        <img id="triangle" className="effects" src="/triangle.svg" alt="Visual Effect"/>
+      </div>
+
+      {/* style={{marginLeft: ((i-34)*(30)).toString() + 'px'}} */}
+
+      <div id="middle">
+       {keyboard.map((d,i) => {
+         if(d !== d.toString().toUpperCase()) return <div onMouseDown={() => play({ id: d.toString() })} id='key'><span>{d}</span></div>
+         else return <div onMouseDown={() => play({ id: d.toString() })} style={{marginLeft: ((((i-(18 * 4))*2)*30)).toString() + 'px'}} id='black_key'/>
```

Comment:
The equation needs to be improved, the black keys are not in the correct position.🙁